cartographer/mapping/pose_extrapolator_interface.cc

```diff
@@ -51,6 +51,7 @@ PoseExtrapolatorInterface::CreateWithImuData(
     const proto::PoseExtrapolatorOptions& options,
     const std::vector<sensor::ImuData>& imu_data) {
   CHECK(!imu_data.empty());
+  CHECK(!options.use_imu_based()) << "Not implemented!";
```

Comment:
Add a `TODO` that the intent is to implement this?

Reply:
done